src/main/scala/com/exasol/cloudetl/kafka/deserialization/DeserializationFactory.scala

```diff
@@ -47,8 +48,11 @@ object DeserializationFactory {
       case "string" => StringDeserialization
       case _ =>
         throw new KafkaConnectorException(
-          s"Unsupported format '$format'. " +
-            s"Valid values are [avro, json, string]"
+          ExaError
+            .messageBuilder("E-KCE-19")
+            .message("The format {{FORMAT}} is not supported.", format)
+            .mitigation("Please use one of 'avro', 'json' or 'string' formats.")
```

Comment:
No sure, maybe you can also add the supported ones as a list. Then new one would automatically be added